### PR TITLE
Split test and default actions

### PR DESCRIPTION
**Pull request by achamayou**

### CODE REVIEW

File: tests/governance_js.py [hunk 1]
```diff
@@ -345,7 +345,7 @@ def test_actions(network, args):
         r = c.post("/gov/proposals.js", valid_set_member_data)
         assert r.status_code == 200, r.body.text()
 
-        valid_rekey_ledger = proposal(action("rekey_ledger"))
+        valid_rekey_ledger = proposal(action("trigger_ledger_rekey"))
```
Comment (achamayou) [inline tests/governance_js.py:348]: Minor drive-by rename to match what we specc'ed in https://github.com/microsoft/CCF/discussions/2169#discussioncomment-373490

File: src/runtime_config/default_actions.js [hunk 2]
```diff
@@ -250,32 +168,11 @@ const actions = new Map([
...
-      function (args) {
-        return true;
-      },
-      function (args) {
-        throw new Error("Error message");
-      }
-    ),
-  ],
   [
     "set_ca_cert_bundle",
```
Comment (achamayou) [inline src/runtime_config/default_actions.js:172]: Note, this doesn't quite match the spec, where we didn't have _bundle: https://github.com/microsoft/CCF/discussions/2169#discussioncomment-373490

I've left it as is because it seems reasonable anyway, @letmaik feel free to correct the other way if you prefer.

Comment (letmaik) [reply src/runtime_config/default_actions.js:172]: I've changed it in the spec.

**Review by eddyashton — APPROVED** [review]
